applications/FreeSurfaceApplication/custom_utilities/edge_data_c2c.h

```diff
@@ -0,0 +1,1374 @@
+/*
```

Comment:
I don't remember why we have created this version for C2C on that time and what was the difference. @RiccardoRossi and @antonialarese do you think we should keep it?